### PR TITLE
add the new event docs

### PR DESCRIPTION
**Pull request by shubhika03**

### CODE REVIEW

File: _events/STAX.md [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+
+    * Dropping the block each time: **-30** (L)
+
+    * For each timeout: **-100** (O)
+
+    * For each restart: **-150** (S)
+
+Formula:
+
+1000 + 100*(Q) + 150*(P + R) + 2*T - 30*L - 50*A - 100*O - 150*S
```
Comment (narayanaditya95) [inline _events/STAX.md]: Try syntax highlighting:

```ada
1000 + 100*(Q) + 150*(P + R) + 2*T - 30*L - 50*A - 100*O - 150*S
```

Use:

```
{% highlight ada %}
1000 + 100*(Q) + 150*(P + R) + 2*T - 30*L - 50*A - 100*O - 150*S
{% endhighlight %}
```

File: _events/POLES APART.md [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+title: 'Poles Apart'
+subTitle: 'Manual Robotics Event'
+logo: /img/event/polesapart/image.jpg
+---
+			
+
+
+**Problem Statement:**
+
+To build a manually controlled robot, which is capable of **picking and placing blocks** with accuracy and** changing its interaxial distance** to make a guide through a series of hurdles.
```
Comment (narayanaditya95) [inline _events/POLES APART.md]:
```
and **changing
```

Italics/bold `**` must start just before a word.

File: _events/POLES APART.md [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+
+* Placing blocks in their respective places.
+
+**Task:**
+
+**Round 1:**
+
+* Complete its way up a slope by *picking blocks from an adjacent area* and *placing them in their respective slot.*
+
+* The* Block 1, Block 2 and Block 3 are in increasing order of their height up the slope* and have different points for each.
```
Comment (narayanaditya95) [inline _events/POLES APART.md]:
```
The blocks 1, 2 and 3 ...
```

File: _events/POLES APART.md [hunk 4]
```diff
@@ -0,0 +1,260 @@
...
+
+* Place 2 horizontal planks on pillars to make a bridge and cross a valley.
```
Comment (narayanaditya95) [inline _events/POLES APART.md]: \- `2`
\+ `two`

File: _events/POLES APART.md [hunk 5]
```diff
@@ -0,0 +1,260 @@
...
+
+**Top View:**![image alt text](/img/event/polesapart/image_1.png){:.img-responsive}
+
+**Arena specifications:**
+
+![image alt text](/img/event/polesapart/image_2.png){:.img-responsive}
+
+![image alt text](/img/event/polesapart/image_3.png){:.img-responsive}
+
+**Isometric ****View****:**![image alt text](/img/event/polesapart/image_4.png){:.img-responsive}
```
Comment (narayanaditya95) [inline _events/POLES APART.md]: Why bold around the word View?

**Comment by pranitbauva1997** [comment]
@shubhika03 It might also be worthwhile to checkout [pandoc](https://pandoc.org/) to convert documents from one format to another . I used them heavily last year and at the end, it just requires very tiny manual tweaking.

File: _events/stax.md [hunk 6]
```diff
@@ -0,0 +1,266 @@
...
+#### Arena
+
+![](/img/event/stax/image_2.png){:.img-responsive}
+                                     
+
+##### Arena specifications 
+
+* The outer dimensions of the arena will be **505 cm x 360 cm**.
+
+* Each stacks will be a rectangle of inner dimensions **80 cm x 20 cm** with a width of **10 cm** on all sides.
```
Comment (narayanaditya95) [inline _events/stax.md]: `Each stack`

File: _events/fortress.md [hunk 7]
```diff
@@ -0,0 +1,230 @@
...
+
+    * Round 2: **2**
+
+    * Maximum duration of a timeout is 2 mins only.
+
+* Restarts will be given only for technical fault in the bot
+
+**Robot Specifications:**
+
+* The robot must fit in a box of **25cm x 25cm x 25cm** with a tolerance of 10% in the dimensions of robot. No part/mechanism of/on the bot should exceed the given dimensions before the commencement of the event. However the dimensions can change during the course of the run.
```
Comment (narayanaditya95) [inline _events/fortress.md]: `of the robot`

Comment (narayanaditya95) [reply _events/fortress.md]: `However, `

File: _events/stax.md [hunk 8]
```diff
@@ -0,0 +1,266 @@
...
+
+**NOTE:** Restarts will be given only for technical fault in the bot.
+
+##### Scoring Formula:
+
+* **Positives**
+
+    * Base score: **1000**
+
+    * For each blocks picked/gripped(given once for each block): **100** (Q)
```
Comment (narayanaditya95) [inline _events/stax.md]: `For each block`

File: _events/stax.md [hunk 9]
```diff
@@ -0,0 +1,266 @@
...
+
+**NOTE**: Restarts will be given only for technical fault in the bot.
+
+##### Scoring Formula:
+
+* **Positives**
+
+    * Base score: **1000**
+
+    * For each blocks picked/gripped(given once for each block): **100** (Q)
```
Comment (narayanaditya95) [inline _events/stax.md]: `For each block`

File: _events/fortress.md [hunk 10]
```diff
@@ -0,0 +1,230 @@
...
+
+* Dimensions of the arena are to be considered with a maximum tolerance of 10%.
+
+* A marker of dimension **20cm x 10cm** having two colour scheme will be provided. However, the participants are free to use their own markers.
+
+#### Task:
+
+##### Round 1
+
+The arena is plain black with some shapes of same color on it. There would be an overhead camera which would provide a video feed to the bot. In the arena all the shapes will be made of a specific number of squares and will have the same area.
```
Comment (narayanaditya95) [inline _events/fortress.md]: Make all instances of `color` -> `colour`

File: _events/fortress.md [hunk 11]
```diff
@@ -0,0 +1,230 @@
...
+
+##### Round 1
+
+The arena is plain black with some shapes of same color on it. There would be an overhead camera which would provide a video feed to the bot. In the arena all the shapes will be made of a specific number of squares and will have the same area. 
+
+ The participants will also be given an image which will comprise of the same number of shapes but they will be of different sizes and different colours.The shapes will have the same orientation in both the image and the arena.
+
+The task of round one is to identify the  shapes present in the arena corresponding to the shapes in the image by performing pattern recognition. The colour of each shape in the image represents a serial number of that shape and the bot has to go to each shape in the ascending order of serial number. When it identifies that certain shape in the arena it has to go to that shape and has to blink the LED the number of times equal to the serial number of that shape. This round will last for 4 minutes.
+
+**Note:**The serial number corresponding to a colour will be provided at the start of the run. And the RGB values of the colours of the shapes in the images will also be provided.
```
Comment (narayanaditya95) [inline _events/fortress.md]: Space after `**Note**`

File: _events/polesapart.md [hunk 12]
```diff
@@ -0,0 +1,260 @@
...
+
+* Dragging a block or plank across the arena: **-25** (E4)
+
+* Negative marks for each timeout: **-50** (T)
+
+* Negative marks for restart: **-100** (R)
+
+**Scoring formula:**
+
+{% highlight ada %}
```
Comment (narayanaditya95) [inline _events/polesapart.md]: Looks like highlighting with `ada` doesn't work. Make it `python` or `ruby`.

File: _events/fortress.md [hunk 13]
```diff
@@ -0,0 +1,230 @@
...
+title: 'Fortress'
+subTitle: 'Computer Vision Event'
+logo: /img/event/fortress/image.jpg
+---
+
+#### Introduction
+
+A nuclear war just affected our planet. And the robots are now trying to save the remaining humans in a habitable environment by building an impenetrable fortress. For doing this they have to collect all the suitable blocks in proper order in a given time before they lose any more lives.
+
+#### USP:
```
Comment (narayanaditya95) [inline _events/fortress.md]: I would not prefer a `:` in the header.
P.S Everywhere.

File: _events/stax.md [hunk 14]
```diff
@@ -0,0 +1,266 @@
...
+
+* The bot has to **return** to the node in front of the main stack and **indicate the completion of job**.
+
+##### Round rules:
+
+* A maximum of **6 minutes** will be given to complete this round. The restart will only be given before **4:30** into the round.
+
+* Maximum number of Time-outs allowed :**2(1 min each)**
+
+* Restarts allowed :**2**
```
Comment (narayanaditya95) [inline _events/stax.md]: Space before the colon
`allowed: `

File: _events/polesapart.md [hunk 15]
```diff
@@ -0,0 +1,260 @@
...
+
+* Place two horizontal planks on pillars to make a bridge and cross a valley.
+
+* Cross another path by using 2 planks which will already be placed over it. 
+
+* Cross a path which consists of two planks placed at an angle to each other, such that the distance between them changes continuously in a diverging pattern.
+
+* Cross a path which consists of two planks placed at an angle to each other such that the distance between them changes continuously in a converging pattern.
+
+* Make its way down a slope made of two planks .
```
Comment (narayanaditya95) [inline _events/polesapart.md]: Remove space before `.`

File: _events/polesapart.md [hunk 16]
```diff
@@ -0,0 +1,260 @@
...
+
+* Cross a path with two circular planks of constant radius making an arc.
+
+* Make its way up a slope made of two planks which will be at an angle to each other in a converging pattern.
+
+* Make its way down a slope made of two planks which will be at an angle to each other in a converging pattern.
+
+* Make its way down a normal slope to the end.
+
+**Note:** If at any point the bot drops from the arena, a penalty of 50 percent of the existing points will be made, if the bot wishes to start from the same point. Otherwise there is an option for rerun of the bot.
```
Comment (narayanaditya95) [inline _events/polesapart.md]: `Otherwise, there is an option for a rerun of the bot.`

File: _events/polesapart.md [hunk 17]
```diff
@@ -0,0 +1,260 @@
...
+
+**Negatives**
+
+* Dropping a block: **-25** (E1)
+
+* Dropping a plank: **-50** (E2)
+
+* Disbalance of slipping of the robot: **-100** (E3)
+
+* Robot falling of the arena: **50% of the current score**
```
Comment (narayanaditya95) [inline _events/polesapart.md]: `Falling off`

File: _events/polesapart.md [hunk 18]
```diff
@@ -0,0 +1,260 @@
...
```
Comment (narayanaditya95) [inline _events/polesapart.md]: `falling off`

**Comment by narayanaditya95** [comment]
Can you also remove [this file](https://github.com/Robotix/Robotix.github.io/blob/master/Mech.jpg)

**Comment by narayanaditya95** [comment]
Also remove the folder `assets/event/`

**Comment by narayanaditya95** [comment]
Some more changes required:
- Change all reference to Robotix 2017 in _config.yml
- Change "Event Schedule" button on hero to "Explore Events"
- Change stats in statistic pane of homepage
- Remove all FAQs

File: _events/fortress.md [hunk 19]
```diff
@@ -0,0 +1,228 @@
...
+
+ The participants will also be given an image which will comprise of the same number of shapes but they will be of different sizes and different colours.The shapes will have **different** orientation in  the arena than the image. The obstacles will not be present in the given image.
+
+The task of round two is to identify the  shapes present in the arena corresponding to the shapes in the image by performing pattern recognition. The colour of each shape in the image represents a serial number of that shape and the robot has to go to each shape in the ascending order of serial number. When it reaches a shape in the arena it has to blink the LED the number of times equal to the serial number of that shape. The robot has to avoid going over the obstacles in the arena or identifying them as one of the shapes in the image. This round will last for 4 minutes.
+
+**Note:** The serial number corresponding to a colour will be provided at the start of the run. And the RGB values of the colours of the shapes in the images will also be provided.
+
+.
```
Comment (narayanaditya95) [inline _events/fortress.md]: Remove this line.

File: _events/fortress.md [hunk 20]
```diff
@@ -0,0 +1,228 @@
...
+
+* Image segmentation
+
+* Autonomous traversal
+
+#### Problem Statement
+
+Build an image processing robot that can recognise useful patterns by pattern recognition while avoiding other obstacles. The video feed of the arena will be given by an overhead camera.
+
+#### Rules
```
Comment (narayanaditya95) [inline _events/fortress.md]: Rules are written just before the scoring.

File: _events/fortress.md [hunk 21]
```diff
@@ -0,0 +1,228 @@
...
+
+    * Round 1: **2**
+
+    * Round 2: **2**
+
+    * Maximum duration of a timeout is 2 mins only.
+
+* Restarts will be given only for technical fault in the robot
+
+**Robot Specifications:**
```
Comment (narayanaditya95) [inline _events/fortress.md]: `Robot Specification` should be a header in itself, just before `Tasks`.

File: _events/fortress.md [hunk 22]
```diff
@@ -0,0 +1,228 @@
...
+
+ The participants will be given an image which will comprise of the same number of shapes as the arena but they will be of different sizes and different colours. The shapes will have the same orientation in both the image and the arena.
+
+The task of round one is to identify the  shapes present in the arena corresponding to the shapes in the image by performing pattern recognition. The colour of each shape in the image represents a serial number of that shape and the robot has to go to each shape in the ascending order of serial number. When it reaches a shape in the arena it has to blink the LED the number of times equal to the serial number of that shape. This round will last for 4 minutes.
+
+**Note:** The serial number corresponding to a colour will be provided at the start of the run. The RGB values of the colours of the shapes in the image will also be provided.
+
+#### Arena
+
+**Round 1:**
```
Comment (narayanaditya95) [inline _events/fortress.md]: Remove this line

File: _events/poles-apart.md [hunk 23]
```diff
@@ -0,0 +1,260 @@
...
+
+* Make its way down a normal slope to the end.
+
+**Note:** If at any point the robot drops from the arena, a penalty of 50 percent of the existing points will be made, if the robot wishes to start from the same point. Otherwise, there is an option for rerun of the robot.
+
+#### Arenas
+
+##### Round 1
+
+**Top View:**![](/img/event/polesapart/image_1.png){:.img-responsive}
```
Comment (narayanaditya95) [inline _events/poles-apart.md]: Image in the next line.

File: _events/poles-apart.md [hunk 24]
```diff
@@ -0,0 +1,260 @@
...
+
+**Arena specifications:**
+
+![](/img/event/polesapart/image_2.png){:.img-responsive}
+
+![](/img/event/polesapart/image_3.png){:.img-responsive}
+
+**Isometric View:**![](/img/event/polesapart/image_4.png){:.img-responsive}
```
Comment (narayanaditya95) [inline _events/poles-apart.md]: Image in the next line

**Comment by narayanaditya95** [comment]
@shubhika03 Pushed to master locally. Thank you for the PR. Good work.

**Pull request closed without merging**